lua/smh/client/render.lua

```diff
@@ -51,4 +50,11 @@ function SMH.StopRender()
 
 	SMH.Rendering = false;
 
+end
+
+function SMH.FinishRender()
```

Comment:
This could be done in `StopRender` instead to avoid making another function. Checking the value of `SMH.Data.Rendering` first should prevent a loop.

So in `StopRender` after `SMH.Rendering = false`, you can add something like this:
```
if SMH.Data.Rendering then
    SMH.Data.Rendering = false;
end